src/middlewared/middlewared/plugins/smb.py

```diff
@@ -268,6 +286,30 @@ def key_cp(encoding):
             ] + initial
         }
 
+    @private
+    @periodic(interval=86400)
+    async def do_smb_alerts(self):
+        await self.unix_extensions_alert()
+        await self.middleware.call('sharing.smb.share_configuration_alert')
+
+    @private
+    async def unix_extensions_alert(self):
+        smb_running = await self.middleware.call('service.started', 'cifs')
+        if not smb_running:
+            del_job = await self.middleware.call('alert.oneshot_delete', 'UnsupportedUnixExtensions', None)
+            await del_job.wait()
```

Comment:
Why wait for this? It might involve sending alerts, which might involve networking, which might take a long time.

Reply:
Good point. I didn't think that through.